src/libcontentfeed-version.h.in

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2018 Endless Mobile, Inc. */
+
+#pragma once
+
+#if !(defined(LIBCONTENT_FEED_H_INSIDE) || defined(COMPILING_LIBCONTENTFEED))
+#error "Please do not include this header file directly."
+#endif
+
+/**
+ * SECTION:version-information
+ * @title: Version Information
+ * @short_description: Variables and functions to check the library version
+ *
+ * Some macros to check the version of the content library.
+ */
+
+/**
+ * EOS_DISCOVERY_FEED_MAJOR_VERSION:
+ *
+ * Major version number (API level) of the libcontentfeed library.
+ * For example, this is 0 for version 0.6.1.
+ */
+#define EOS_DISCOVERY_FEED_MAJOR_VERSION (@EOS_DISCOVERY_FEED_MAJOR_VERSION@)
```

Comment:
These are not named correctly.

Reply:
I agree, having it named this way sucks but renaming them to `LIBCONTENTFEED` tripped up the introspection process because they weren't part of the symbol prefix. I'm not particularly interested in churning through several thousand lines of code to change a prefix at this point :/